libs/langchain/langchain/chat_models/databricks.py

```diff
@@ -0,0 +1,195 @@
+import asyncio
+import logging
+from functools import partial
+from typing import Any, Dict, List, Mapping, Optional
+
+from langchain_core.messages import (
+    AIMessage,
+    BaseMessage,
+    ChatMessage,
+    FunctionMessage,
+    HumanMessage,
+    SystemMessage,
+)
+from langchain_core.outputs import (
+    ChatGeneration,
+    ChatResult,
+)
+from langchain_core.pydantic_v1 import BaseModel, Extra
+
+from langchain.callbacks.manager import (
+    AsyncCallbackManagerForLLMRun,
+    CallbackManagerForLLMRun,
+)
+from langchain.chat_models.base import BaseChatModel
+
+logger = logging.getLogger(__name__)
+
+
+# Ignoring type because below is valid pydantic code
+# Unexpected keyword argument "extra" for "__init_subclass__" of "object"  [call-arg]
+class ChatParams(BaseModel, extra=Extra.allow):  # type: ignore[call-arg]
+    """Parameters for the `Databricks` LLM."""
+
+    temperature: float = 0.0
+    n: int = 1
+    stop: Optional[List[str]] = None
+    max_tokens: Optional[int] = None
+
+
+class ChatDatabricks(BaseChatModel):
```

Comment:
This `ChatDatabricks` looks almost identical to `ChatMlflow`. Maybe I am missing some context but why do we need this one? Why not just have one `ChatMlflow`?

Reply:
Fixed!